libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFSalesforceAction.m

```diff
@@ -44,10 +49,36 @@ - (instancetype)initWithResponseBlock:(CSFActionResponseBlock)responseBlock {
         _apiVersion = CSFSalesforceDefaultAPIVersion;
         _pathPrefix = CSFSalesforceActionDefaultPathPrefix;
         self.authRefreshClass = [CSFSalesforceOAuthRefresh class];
+        CSFNetwork *network = self.enqueuedNetwork;
+        [network addObserver:self forKeyPath:@"credentials.accessToken"
```

Comment:
The key path for these will need to be e.g. `account.credentials.accessToken`, yes?